draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -181,6 +184,8 @@ separately.
 
 DNS API clients can use HTTP/2 padding and compression in the same way
 that other HTTP/2 clients use (or don't use) them.
+(Note that there is also DNS padding defined in {{RFC7830}} that
```

Comment:
I think we should remove this new text. This section is about how the HTTP request is formed (its not generally about padding strategies) and we elsewehere say we can carry anything you can put in a dns query.